portfolio/src/main/webapp/style.css

```diff
@@ -1,21 +1,27 @@
+/* Top margin for navbar and off-white background */
```

Comment:
1) Fix indentation for whole file: each attribute should be indented 2 spaces.
2) Fix ordering of attributes in each group so that either A) they are grouped logically (i.e. top, left, right, bottom attributes are kept together without other attributes in between them) or B) they are sorted alphabetically.
3) Group CSS selectors by either relevance to each other, type (i.e. foo, .bar, #baz), or some other preferred order (i.e. order they appear in the HTML).

Reply:
I've fixed the indentation to be 2 spaces instead and sorted attributes alphabetically at Longjun's recommendation. Can you go over what you mean by "relevance" in the 3rd point? I tried to group selectors by the order they appear in the HTML for the most part, with comments sectioning off each of these.

Reply:
That's fine. I just wasn't sure what the general ordering was, but now that you've explained it SGTM.